docker/php-7.4/Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM php:7.4-cli-alpine3.16
 ARG DOCKER_USER_ID
 ARG DOCKER_GROUP_ID
 
+# https://blog.codito.dev/2022/11/composer-binary-only-docker-images/
+COPY --from=composer/composer:2-bin /composer /usr/local/bin/composer
```

Comment:
If you guys don't like self-promotion like this, we can link composer/docker#250 instead 😉. I just wanted to provide background.